pkgs/development/libraries/science/networking/ns-3/default.nix

```diff
@@ -97,6 +97,10 @@ stdenv.mkDerivation rec {
     ${pythonEnv.interpreter} ./test.py --nowaf
   '';
 
+  patches = [
+    ./upstream-issue-336.patch # See https://gitlab.com/nsnam/ns-3-dev/-/issues/336
```

Comment:
They solved the issue differently upstream. Do we want to fetchpatch https://gitlab.com/nsnam/ns-3-dev/-/commit/673004edae1112e6cb249b698aad856d728530fb.patch instead?

Reply:
Yeah, there was two way of solving this issue in the thread, I'll rewrite the change.